layouts/partials/fragments/list.html

```diff
@@ -36,7 +36,9 @@
       <div class="row mx-0
         {{- partial "helpers/text-color.html" (dict "self" $self.self "light" "secondary") -}}
       ">
+        {{ partial "helpers/slot.html" (dict "root" $ "slot" "before-all")}}
         {{- range first (.Params.count | default 10) (.page_scratch.Get "pages") -}}
+          {{ partial "helpers/slot.html" (dict "root" $ "slot" "before-item")}}
```

Comment:
Why the specific `before-item`? What's the use case?

Reply:
It can be used if the user wants to display author name, categories or any other meta data before an article.